pig/src/main/java/org/jboss/pnc/bacon/pig/impl/pnc/PncBuilder.java

```diff
@@ -60,13 +60,16 @@ public PncBuilder() {
         groupConfigClient = gc;
     }
 
-    public GroupBuild buildAndWait(
+    public GroupBuild buildAndMaybeWait(
             GroupConfigurationRef group,
             boolean tempBuild,
             boolean tempBuildTS,
-            RebuildMode rebuildMode) {
+            RebuildMode rebuildMode,
+            boolean noWait) {
         GroupBuild groupBuild = run(group, tempBuild, tempBuildTS, rebuildMode);
-        waitForSuccessfulFinish(groupBuild.getId());
+        if (!noWait) {
```

Comment:
just a note, I think it would be a bit clearer if the method param was the inverse, `wait`, and not `noWait`